src/DistributedTracingManager.ts

```diff
@@ -0,0 +1,251 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as vscode from "vscode";
+import { BaseExplorer } from "./baseExplorer";
+import { Constants, DistributedSettingUpdateType } from "./constants";
+import { TelemetryClient } from "./telemetryClient";
+import { Utility } from "./utility";
+import iothub = require("azure-iothub");
+import uuid = require("uuid");
+import { DeviceItem } from "./Model/DeviceItem";
+import { SamplingModeItem } from "./Model/SamplingModeItem";
+import { DistributedTracingLabelNode } from "./Nodes/DistributedTracingLabelNode";
+import { DistributedTracingSettingNode } from "./Nodes/DistributedTracingSettingNode";
+
+export class DistributedTracingManager extends BaseExplorer {
+    constructor(outputChannel: vscode.OutputChannel) {
+        super(outputChannel);
+    }
+
+    public async updateDistributedTracingSetting(node, updateType?: DistributedSettingUpdateType) {
```

Comment:
line 27 - 29 could be removed: `updateType?: DistributedSettingUpdateType` -> `updateType: DistributedSettingUpdateType = DistributedSettingUpdateType.All`